fixtures/fragments/file1.md

```diff
@@ -66,7 +66,7 @@ without related HTML element. Browser will scroll to the top of the page.
 
 A link to the non-existing fragment: [try](https://github.com/lycheeverse/lychee#non-existent-anchor).
 
-Skip the fragment check for directories like: [empty](empty_dir/).
+A link to a fragment in an empty directory: [empty](empty_dir/#fragment).
```

Comment:
Should we keep the old link as well to clearly specify the behavior?

```suggestion
Skip the fragment check for directories like: [empty](empty_dir/).
A link to a fragment in an empty directory: [empty](empty_dir/#fragment).
```

Reply:
Let me add some more links to better describe the current behavior

Reply:
added in https://github.com/lycheeverse/lychee/pull/1756/commits/7e0b8d444df3c6e94ca6eb9c988f8ef80744c206